### ISSUE TITLE
Elements must meet minimum color contrast ratio

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1141]
for the blue buttons: Element has insufficient color contrast of 3.55 (foreground color: #ffffff, background color: #0087ff, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1

Possible solution:
background color:

- #0075DB ( works for WCAG AA)
- #0057A3  ( works for WCAG AAA)
